core/src/test/java/ai/timefold/solver/core/testdomain/mixed/singleentity/MixedCustomBasicVariableSwapMove.java

```diff
@@ -0,0 +1,31 @@
+package ai.timefold.solver.core.testdomain.mixed.singleentity;
+
+import ai.timefold.solver.core.api.score.director.ScoreDirector;
+import ai.timefold.solver.core.impl.heuristic.move.AbstractMove;
+
+public class MixedCustomBasicVariableSwapMove extends AbstractMove<TestdataMixedSolution> {
```

Comment:
Why do we even need this? Looks like a simple swap move could have been reused. There is no special logic here.